plugins/obs-ffmpeg/obs-ffmpeg-mpegts.c

```diff
@@ -608,10 +603,13 @@ static void close_mpegts_url(struct ffmpeg_output *stream, bool is_rist)
 	av_freep(h);
 
 	/* close custom avio_context for srt or rist */
-	avio_flush(stream->s);
-	stream->s->opaque = NULL;
-	av_freep(&stream->s->buffer);
-	avio_context_free(&stream->s);
+	AVIOContext *s = stream->s;
+	if (!s)
+		return;
+	avio_flush(s);
```

Comment:
Put a newline between the previous `if` contents and this, or enclose the previous `if` contents in curly braces.

Reply:
done